assets/styles/options.css

```diff
@@ -176,13 +182,27 @@ body {
 }
 
 #whiteList_domains_table {
-    display: none;
     margin-block: 1.5em;
     min-width: 100%;
 }
 
-#whiteList_domains td {
+.whitelist-ent {
+    margin-block: 0.125rem;
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    grid-auto-flow: row;
+}
+
+.whitelist-ent>.whitelist-txt {
+    grid-column: 1 / span 10;
+    word-wrap: break-word;
     font-size: 1.2rem;
     font-weight: 400;
     padding: 6px 0;
+
+}
+.whitelist-ent>.whitelist-del-container {
+    grid-column: auto / span 2;
+    display: grid;
+    place-items: center;
```

Comment:
These actions should be performed with media queries to make the page responsive.

Reply:
those css lines makes it reponsive without media queries. Check and let me know if anything else is required.

Reply:
> those css lines makes it reponsive without media queries. Check and let me know if anything else is required.

Ok - will look into it.

Reply:
@tusharsnn I am going to reject this PR as it doesn't seem to fix the problem - it should make the item 100% width on small screens.